### DISCUSSION TITLE
Elgato wave 3, Geforce expierence.

### DISCUSSION THREAD

**Discussion by VinceNotVincent** [elgatosf/elgato#33] [Help]
When volume is changed on elgato wave 3, I understand it changes the volume in device manager as expected. But when it happens, it also changes mic volume in geforce expierence. Ex: If I change gain to 60%, in geforce it makes the volume 60% of that 60%.

**Comment by ChekTek** [comment]
For general Elgato support, please reach out through http://e.lga.to/help.
